cmake/cmake_install.cmake

```diff
@@ -0,0 +1,60 @@
+# Install script for directory: /home/san/somecode/satellogic/gr-dvb
```

Comment:
Please make this generic without hardcoded file paths for a particular environment "/home/san/somecode/satellogic/gr-dvb"